.github/ISSUE_TEMPLATE/icon_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue, please
+        search for duplicate or closed issues or PRs. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction to the original
+        post.
+
+  - type: input
+    attributes:
+      label: Brand Name
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Alexa Rank
+      description: >-
+        The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/. Please see
+        our contributing guidelines for more details on how we assess a brand's
+        popularity.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
+      description: >-
+        Media kits, brand guidelines, SVG files, etc. For more details on our
+        processes please see our contributing guidelines, which can be found at
+        https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md.
```

Comment:
Noticing now that including a link to the contributing guidelines in the "Alexa rank" input as well is a bit of duplication. I wonder if we can instead move this to "the top" of the form :thinking:

Reply:
That would make more sense as it's not specific to the brand resources and color anyway